### PR TITLE
[Kotlin Migration] CopyrightHeaderExistsTest Lint TEST

### PR DESCRIPTION
**Pull request by prateek-singh-3212**
Migrated CopyrightHeaderExistsTest.java to CopyrightHeaderExistsTest.kt

### CODE REVIEW

File: lint-rules/src/test/java/com/ichi2/anki/lint/rules/CopyrightHeaderExistsTest.kt [hunk 1]
```diff
@@ -13,74 +13,70 @@
...
     @Test
-    public void fileWithCopyrightHeaderPasses() {
-        lint()
-                .allowMissingSdk()
-                .files(create(mCopyrightHeader))
-                .issues(CopyrightHeaderExists.ISSUE)
-                .run()
-                .expectClean();
+    fun fileWithCopyrightHeaderPasses() {
+        TestLintTask.lint()
```
Comment (david-allison) [inline lint-rules/src/test/java/com/ichi2/anki/lint/rules/CopyrightHeaderExistsTest.kt:60]: Could do with fixing the imports to match the Java:

`TestLintTask`, `JavaTestFile`, `Assert`

**Review by david-allison — CHANGES_REQUESTED** [review]
Minor fix, then looks good to go, thanks!

File: lint-rules/src/test/java/com/ichi2/anki/lint/rules/CopyrightHeaderExistsTest.kt (hunk 1 above)
Comment (prateek-singh-3212) [reply lint-rules/src/test/java/com/ichi2/anki/lint/rules/CopyrightHeaderExistsTest.kt]: kindly check

**Review by david-allison — APPROVED** [review]

**Review by mikehardy — APPROVED** [review]
another one converted :muscle:

**Pull request merged**